content/blog/2019-11-15-the-state-of-r-in-africa.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The-state-of-R-in-Africa
+banner:"/images/blog/The-state-of-R-in-Africa/banner.jpg"
+author: Lorna Maria
+date: '2019-11-15'
+description: "2019 Round-up of initiatives in the R community across Africa."
+slug: the-state-of-r-in-africa
+categories: ["news"]
+tags: ["community"]
+tocify: no
+---
+##Growth of the R ecosystem in Africa 
```

Comment:
```suggestion
## Growth of the R ecosystem in Africa 
```

Reply:
spaces are needed after `##` otherwise the title is not recognised when the markdown is rendered.